app/Models/TnmsAlarms.php

```diff
@@ -6,6 +6,6 @@
 
 class TnmsAlarm extends Model
 {
-    //
+    protected $table = 'tnmsalarms';
```

Comment:
If the table is tnms_alarms, you don't need this :)

Reply:
Yeah, following the standard is preferred.  Can't do anything about existing tables.